components/form/index.js

```diff
@@ -0,0 +1,126 @@
+import { CheckCircle } from 'phosphor-react'
+import { rem } from 'polished'
+import { useEffect, useState } from 'react'
+import styles from 'styles/Home.module.css'
+import Button from '../button'
+import Header from '../header'
+import Input from '../input'
+import Text from '../text'
+import TextArea from '../textarea'
+
+const Form = () => {
+  const [isFormSubmitted, setIsFormSubmitted] = useState(false)
+  const [hasErrors, setHasErrors] = useState(false)
+  const [state, setState] = useState({
+    email: '',
+    info: '',
+    name: '',
+  })
+
+  const onChange = (e) => {
+    setState({ ...state, [e.target.name]: e.target.value })
+  }
+
+  const onClick = async (e) => {
+    e.preventDefault()
+    setHasErrors(false)
+    try {
+      const res = await fetch(process.env.NEXT_PUBLIC_HELIX_HOST, {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(state),
+      })
+      setIsFormSubmitted(true)
+      return res.json()
+    } catch (err) {
+      return setHasErrors(true)
+    }
+  }
+
+  useEffect(() => {
+    setIsFormSubmitted(localStorage.getItem('formSubmitted'))
+  }, [])
+
+  useEffect(() => {
+    localStorage.setItem('formSubmitted', isFormSubmitted)
+  }, [isFormSubmitted])
+
+  const renderForm = () =>
+    isFormSubmitted === true ? (
+      <div>
+        <CheckCircle
+          {...{
+            color: '#FF68BA',
+            size: rem('37px'),
+            weight: 'fill',
+          }}
+        />
+        <Header {...{ successTitle: true }}>Application Sent !</Header>
+        <Text successText>
+          Look out for your inbox! Someone from Commit will be in touch with
```

Comment:
`Look out for your inbox!` sounds kinda weird to me but it looks like it's in the mocks